scripts/generate-redstone-data.ts

```
@@ -0,0 +1,231 @@
+    'redstone-data',
+    `${filename}.json`,
+  );
+  writeFileSync(outputPath, JSON.stringify(data, null, 2));
+  console.log(`Written ${data.length} entries to ${filename}.json`);
+};
+
+const main = async (): Promise<void> => {
+  console.log('Starting Redstone oracle data generation...\n');
+
+  try {
+    const networks = Object.keys(ENDPOINTS) as Array<keyof typeof ENDPOINTS>;
```

ESLint

[error] Array type using 'Array<T>' is forbidden. Use 'T[]' instead.

(@typescript-eslint/array-type)